lib/taskana-core/src/main/java/pro/taskana/classification/api/models/Classification.java

```diff
@@ -36,6 +36,27 @@
    */
   String getDomain();
 
+  /**
+   * Get the logical name of the associated application entry point.
+   *
+   * @return applicationEntryPoint
+   */
+  String getApplicationEntryPoint();
+
+  /**
+   * Set the logical name of the associated application entry point.
+   *
+   * @param applicationEntryPoint The application entry point
+   */
+  void setApplicationEntryPoint(String applicationEntryPoint);
+
+  /**
+   * Duplicates this Classification.
```

Comment:
Please comment that the id will not be duplicated.
(and do this for every entity)